src/worldcereal/utils/refdata.py

```diff
@@ -154,7 +262,63 @@ def process_parquet(public_df_raw: pd.DataFrame) -> pd.DataFrame:
     from presto.utils import process_parquet as process_parquet_for_presto
 
     logger.info("Processing selected samples ...")
+
+    if processing_period is not None:
+        logger.info("Aligning the samples with the user-defined temporal extent ...")
+
+        # get the middle of the user-defined temporal extent
+        start_date, end_date = processing_period.to_datetime()
+        processing_period_middle_ts = start_date + pd.DateOffset(months=5)
```

Comment:
the middle is hard to define with an even number of timesteps. Should we be aligned with e.g. [this](https://github.com/WorldCereal/worldcereal-classification/blob/1a394187a3595fd94ddbe1ac7fbc153d1ce79637/src/worldcereal/openeo/feature_extractor.py#L306)? And therefore rather add 6 months? I may be wrong here, was just wondering.